### ISSUE TITLE
Open the consent after it was closed

### ISSUE THREAD

**Issue by Radu033** [nilsenpaul/craft-complete-cookie-consent#31]
Is it possible to open the consent after the user closed it? 
If someone changes their mind on the selected cookies, I would like to add a button at the bottom that will open the consent. Is there a predefined function? I see that currently the consent is removed from the DOM completely upon saving the options.

**Comment by martinsotirov** [comment]
That's essential to my use case as well. For the time being my ugly workaround is to have a button that deletes the consent cookie via JS and reloads the page.

**Comment by nilsenpaul** [comment]
@Radu033 Sorry, your issue/request completely slipped my attention. I think it would be nice to have (at least) a JS function to show the banner. To be honest, though, I think it will be January before we can get this done.
